docs/backup-restore.md

```diff
@@ -3,30 +3,63 @@
+
+You must pause for 10 seconds to create a backup and use the midtime of the pause as a restore point.
+
+## Scalar DB Supported Databases
+
+### Cassandra
 
 Since Cassandra has a built-in replication mechanism, we don't always need a transactionally-consistent backup to recover a Cassandra cluster.
 
 For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, we don't need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
 However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
 
-The easiest way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. If you implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing on-going transactions.
+Follow [How to create transactionally-consistent backup](#how-to-create-transactionally-consistent-backup) to create a transactionally-consistent backup.
+
+Another way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. 
 
 To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
 Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
 Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
 
-## JDBC databases
+### Cosmos DB
+
+You should create Cosmos DB account using `Continuous` backup policy to create continuous backups.
+
+Follow [How to create transactionally-consistent backup](#how-to-create-transactionally-consistent-backup) to create a transactionally-consistent backup.
+
+You can restore the backup based on the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and you must change the default consistency to `STRONG` after restoring the data.
+
+### DynamoDB
+
+You should create the schema using the scalardb schema loader, which enables Point-In-Time-Recovery (PITR) for each table in DynamoDB. It will create continuous backups.
```

Comment:
PITR is a must? It's not clear.
I don't think we should talk about the schema loader here. It's very confusing.

Reply:
Sorry for the inconvenience.
Document updated with the above suggestion.